Wake On Lan/Modules/PostLaunch/Sources/PostLaunchView.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  PostLaunchView.swift
+//
+//
+//  Created by Dmitry Stavitsky on 17.09.2022.
+//
+
+import SnapKit
+import UIKit
+import WOLResources
+
+final class PostLaunchView: UIView {
+    // MARK: - Appearance
+
+    private let appearance = Appearance(); struct Appearance {
+        /// Application main logo
+        let logoImage = Asset.Assets.owl.image
+    }
+
+    // MARK: - Properties
+
+    private lazy var logoImageView: UIImageView = {
```

Comment:
It is awful solution as image does not change according to the current color scheme. It should be fixed in the next release/commit/PR.